Test/WebTestCase.php

```diff
@@ -552,16 +552,14 @@ private function locateResources($paths)
      */
     public function loadFixtureFiles(array $paths = array(), $append = false, $omName = null, $registryName = 'doctrine', $purgeMode = null)
     {
-        if (!class_exists('Nelmio\Alice\Fixtures')) {
-            // This class is available during tests, no exception will be thrown.
-            // @codeCoverageIgnoreStart
-            throw new \BadMethodCallException('nelmio/alice should be installed to use this method.');
-            // @codeCoverageIgnoreEnd
-        }
-
         /** @var ContainerInterface $container */
         $container = $this->getContainer();
 
+        $persisterLoaderServiceName = 'fidry_alice_data_fixtures.doctrine.persister_loader';
+        if (!$container->has($persisterLoaderServiceName)) {
```

Comment:
@theofidry This is always returning false even with the enabled bundle: https://github.com/liip/LiipFunctionalTestBundle/pull/337/files#diff-5658229c21833f6b720a1c07e3705aa0R26

Any idea?

Reply:
I know why: Wrong kernel. :troll: